libpod/runtime_volume_linux.go

```diff
@@ -67,9 +67,20 @@ func (r *Runtime) newVolume(ctx context.Context, options ...VolumeCreateOption)
 	if volume.config.Driver == define.VolumeDriverLocal {
 		logrus.Debugf("Validating options for local driver")
 		// Validate options
-		for key := range volume.config.Options {
-			switch key {
-			case "device", "o", "type", "UID", "GID", "SIZE", "INODES":
+		for key, val := range volume.config.Options {
+			switch strings.ToUpper(key) {
+			case "DEVICE":
+				switch volume.config.Options["type"] {
```

Comment:
Probably want to `ToLower()` this to ensure no case-comparison issues

Reply:
Fixed.